src/PositionManager.sol

```diff
@@ -95,13 +95,17 @@ contract PositionManager is
 
     /// @param unlockData is an encoding of actions, params, and currencies
     /// @param deadline is the timestamp at which the unlockData will no longer be valid
-    function modifyLiquidities(bytes calldata unlockData, uint256 deadline)
+    function unlockAndModifyLiquidities(bytes calldata unlockData, uint256 deadline)
```

Comment:
i think this needs a new name.. lock/unlock is really overloaded also this is the preferred integration path and so when you compare to just `modifyLiquidities` it actually makes this path sound more intense

Reply:
what about this as `modifyLiquidities` and then without the unlock callback its  `modifyLiquiditiesDirect`

Reply:
my vote is `modifyLiquidities` and `modifyWithoutUnlock` -- but agreed the major function shouldn't be `unlockAndModifyLiquidities`